proto/prysm/v1alpha1/attestation.proto

```diff
@@ -126,6 +126,25 @@ message AttestationElectra {
     bytes committee_bits = 4 [(ethereum.eth.ext.ssz_size) = "committee_bits.size", (ethereum.eth.ext.cast_type) = "committee_bits.type"];
 }
 
+message SignedAggregateAttestationAndProofSingle {
+    // The aggregated attestation and selection proof itself.
+    AggregateAttestationAndProofSingle message = 1;
+
+    // 96 byte BLS aggregate signature signed by the aggregator over the message.
+    bytes signature = 2 [(ethereum.eth.ext.ssz_size) = "96"];
+}
+
+message AggregateAttestationAndProofSingle {
+    // The aggregator index that submitted this aggregated attestation and proof.
+    uint64 aggregator_index = 1  [(ethereum.eth.ext.cast_type) = "github.com/prysmaticlabs/prysm/v5/consensus-types/primitives.ValidatorIndex"];
+
+    // The aggregated attestation that was submitted.
+    SingleAttestation aggregate = 3;
+
+    // 96 byte selection proof signed by the aggregator, which is the signature of the slot to aggregate.
```

Comment:
My understanding is that this is required because of the `savePendingAtt` API. It seems annoying to define new proto types just to satisfy an API for the cache `savePendingAtt`. I wonder if there's something we can do to fix the cache for the API; probably a subsequent PR is okay

Reply:
The pending atts queue would have to be redesigned so that you don't have to save `AggregateAttestationAndProof` objects. We currently construct dummy objects just to satisfy the interface.